README.md

```diff
@@ -123,152 +125,244 @@ Congratulations! You can now send remote push notifications to your device from
 
 # Using the SDK
 
-1. ##### Handle Push Notifications
+##### 1. Handle Push Notifications
 
-	When the user taps on the push notification or one of the action buttons, the system calls `UNUserNotificationCenterDelegate`'s [userNotificationCenter:didReceiveNotificationResponse:withCompletionHandler:](https://developer.apple.com/documentation/usernotifications/unusernotificationcenterdelegate/1649501-usernotificationcenter?language=objc). Pass this call to **`IterableAppIntegration`** to track push open event and perform the associated action (see below for custom action and URL delegates).
+When the user taps on the push notification or one of the action buttons, the system calls `UNUserNotificationCenterDelegate`'s [userNotificationCenter:didReceiveNotificationResponse:withCompletionHandler:](https://developer.apple.com/documentation/usernotifications/unusernotificationcenterdelegate/1649501-usernotificationcenter?language=objc). Pass this call to **`IterableAppIntegration`** to track push open event and perform the associated action (see below for custom action and URL delegates).
```

Comment:
- Do you want to link to the Swift-based doc instead?
- Not sure if the `IterableAppIntegration` needs to be bold
- Could you clarify "pass this call to..."?

Reply:
Eagle eyes.